tests/test.py

```diff
@@ -0,0 +1,4 @@
+bla1
```

Comment:
W292 (flake8): no newline at end of file  <!-- Automatically posted by Lintly -->